src/ResourceStateBuilder.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import shortid from 'shortid'
+
+const rdf = require('rdf-ext')
+const _ = require('lodash')
+
+/**
+ * Builds Redux state from an RDF graph
+ */
+export default class ResourceStateBuilder {
+  /**
+   * @param {Dataset} dataset the RDF graph
+   * @param {string} URI of the resource
```

Comment:
add the parameter name

Reply:
e.g., `@param {string} resourceURI - the URI of the resource`

Reply:
Done.